Modules/Common/include/Common/TH1Ratio.h

```diff
@@ -70,11 +70,13 @@ class TH1Ratio : public T, public o2::mergers::MergeInterface
   Bool_t Add(const TH1* h1, const TH1* h2, Double_t c1 = 1, Double_t c2 = 1) override;
   Bool_t Add(const TH1* h1, Double_t c1 = 1) override;
   void SetBins(Int_t nx, Double_t xmin, Double_t xmax) override;
+  void Sumw2(Bool_t flag = kTRUE) override;
 
  private:
   T* mHistoNum{ nullptr };
   T* mHistoDen{ nullptr };
   bool mUniformScaling{ true };
+  Bool_t mSumw2Enabled{ kFALSE };
```

Comment:
```suggestion
  Bool_t mSumw2Enabled{ kTRUE };
```
Should it be also `true` here for consistency with `init()`? Otherwise, someone may look at the header and assume that it is disabled by default, while it is anyway set to true in `init()`.

Reply:
Yes, good point, I'll push the change a bit later.